packages/angular_devkit/build_ng_packagr/src/build/index.ts

```diff
@@ -0,0 +1,44 @@
+import { BuildEvent, Builder, BuilderContext, Target } from '@angular-devkit/architect';
+import { getSystemPath } from '@angular-devkit/core';
+import { Observable } from 'rxjs/Observable';
+import { resolve as resolvePath } from 'path';
+import * as ngPackagr from 'ng-packagr';
+
+// XX: blatantly copy-pasted from 'require-project-module.ts'
+const resolve = require('resolve');
+function requireProjectModule(root: string, moduleName: string) {
+  return require(resolve.sync(moduleName, { basedir: root }));
+}
+
+
+export interface NgPackagrBuilderOptions {
+  project: string;
+}
+
+export class NgPackagrBuilder implements Builder<NgPackagrBuilderOptions> {
+
+  constructor(public context: BuilderContext) { }
+
+  run(target: Target<NgPackagrBuilderOptions>): Observable<BuildEvent> {
+    const root = getSystemPath(target.root);
+    const options = target.options;
+
+    if (!options.project) {
+      throw new Error('A "project" must be specified to build a library\'s npm package.');
+    }
+
+    return new Observable(obs => {
+      const projectNgPackagr = requireProjectModule(root, 'ng-packagr') as typeof ngPackagr;
+      const packageJsonPath = resolvePath(root, options.project);
+
+      projectNgPackagr.ngPackagr()
+        .forProject(packageJsonPath)
+        .build()
+        .then(() => obs.complete())
+        .catch((e: any) => obs.error(e));
```

Comment:
At the moment, the only way is passing a file path in either [`ngPackagr.forProject()`](https://github.com/dherges/ng-packagr/blob/master/src/lib/ng-v5/packagr.ts#L28-L38) or in the [`build()` command](https://github.com/dherges/ng-packagr/blob/master/src/lib/commands/build.command.ts#L4-L22)